src/js/program/media-controller.js

```diff
@@ -231,7 +235,16 @@ export default class MediaController extends Eventable {
 
     set background(shouldBackground) {
         const { container, provider } = this;
-        if (!container || !provider.video) {
+        // A provider without a video tag must use attach and detach
+        if (!provider.video) {
```

Comment:
Do you think it would be better to handle this higher up, and add this logic to `controller.js`?
https://github.com/jwplayer/jwplayer/blob/master/src/js/controller/controller.js#L632

Reply:
That would bring into question every place that we check `Features.backgroundLoading`. So no, I do not.

Reply:
Ok